README.md

```diff
@@ -14,7 +14,7 @@ Patchinko exposes 3 functions: `patch`, `scope`, & `ps`.
 
 If any target properties are instances of `scope(function)`, it will supply the scoped function with the target property for that key, and assign the result back to the target.
 
-`ps` is a composition of `patch` & `scope`, for when you need to patch recursively.
+`ps([ target, ] input)` is a composition of `patch` & `scope`, for when you need to patch recursively. If you supply a `target`, the original value will be left untouched (useful for immutable patching).
```

Comment:
I was misreading the signature last night and interpreted `[ ...targets,] input`, so I couldn't wrap my head around the desired semantics. Got it now! 👍